proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,434 @@
+the maintainers agree that the implementation is not as discussed or there is
+not significant demand or bandwidth to complete it, the issue may go back to the
+intake phase and be available for another user to contribute a change.
+
+If an issue has not been explicitly marked as being available for contribution,
+it should not be expected that a pull request will get a review.
+
+## Measuring success and making adjustments
+
+We acknowledge that the proposed timelines and queue sizes may need to be
+adjusted. We will quantify the process with metrics to determine if it is
+sustainable or if it can be improved. Specifically, we should measure:
+
+1. The number of issues in each state of the contribution process: available for
+   contribution, actively being contributed, ready for review.
+1. The number of popular and highly requested features.
+1. The time elapsed between each state of the contribution process: the start of
+   a contribution and when it is marked as ready for review, the time spent
+   waiting for review, and the time to accept or reject a contribution.
+1. The rate of acceptance and attrition for contributions.
+
+In addition, we will monitor qualitative feedback through comments left on
+issues and pull requests. We will review the feedback for common patterns to
+determine if an improvement to the documentation will alleviate contributor
+frustration. It may also indicate that we need to make changes to the process as
+well.
+
+## Managing the existing backlog
+
+As of 2021-11-02, there are 189 pull requests and 465 issues in the AWS CLI
+GitHub repository. 52 pull requests and 81 issues have not been updated in more
+than a year. 151 issues have upvotes or reactions from five or more users. [TBD]
+of the open pull requests are extremely outdated and would require rebasing,
+which complicates the process of accepting them. [TBD] of the open pull requests
+are directly related to an existing feature request.
+
+Based on these numbers, we propose to close all issues that:
+
+1. Do not have at least 5 upvotes,
+1. Need a rebase that would require significant developer effort, and
+1. Do not have an open issue discussing the proposed change.
+
+To close these issues, we propose an appoach with two phases:
+
+1. We will use existing automated infrastructure to comment on ancient pull
+   requests, defined as no activity in more than a year and less than 5 upvotes.
+   This will post a comment indicating that the issue will be closed without
+   further interaction.
+2. Next, we will review any remaining open pull requests and determine if there
+   is an existing open issue tracking the feature request, and mark all such
+   pull requests for automated closure after [TBD] days. In addition, we will
+   review pull requests that have major conflicts to resolve and manually close
+   them, indicating to the contributor that if they still wish to propose the
+   change they should open an issue for discussion.
+   ## Rationale/FAQ
```

Comment:
Looks like the section header for the `Rationale/FAQ` got merged into this paragraph.

Reply:
fixed